projects/bp-gallery/src/components/views/picture/sidebar/picture-info/TagSelectionField.tsx

```diff
@@ -163,7 +164,8 @@ const TagSelectionField = <T extends TagFields>({
               label={tag.name}
               onClick={() => {
                 window.open(
-                  addNewParamToSearchPath(type, encodeURIComponent(String(tag.name))).searchVal,
+                  addNewParamToSearchPath(useAdvancedSearch ? type : SearchType.ALL, tag.name)
```

Comment:
I am not sure whether its a good idea to lose the `encodeURIComponent` here and also in the respective call in the `TagList.tsx` file. Maybe some tags contain chars like 'ä' or sth like this which can be tricky to display in an URL.
Imo, Instead (to make the ALL-Search work here) we should change the `SearchView.tsx:115` from

```
      const allSearchTerms = searchParams.getAll(SearchType.ALL);
```

to

```
      const allSearchTerms = searchParams.getAll(SearchType.ALL).map(decodeURIComponent);
```

in order to also add a decoding there or what do you think about that? :D